devices.js

```diff
@@ -14230,15 +14230,16 @@ const devices = [
         vendor: 'SONOFF',
         whiteLabel: [{vendor: 'eWeLink', model: 'RHK06'}],
         description: 'Contact sensor',
+        exposes: [e.contact(), e.battery_low(), e.tamper(), e.battery(), e.battery_voltage()],
         fromZigbee: [fz.ias_contact_alarm_1, fz.battery],
         toZigbee: [],
-        meta: {configureKey: 1, battery: {voltageToPercentage: '3V_2500'}},
+        meta: {configureKey: 1},
```

Comment:
Any reason not to bump `configureKey` here?

Reply:
Yes, see the PR comment? 
> The SNZB-04 was annoying, you need to either remove + join or remove the battery for it to take the new reporting values. They also seem to apply when either batteryVoltage or batteryPercentageRemaining come in via attReport. But they don't apply when the ias_contact_alarm is triggered.

It seems the device is reporting already, but forcing everyone to pry them from the location and remove/insert the battery or use a pin to reset them seems like a bad idea.

Reply:
Sorry missed that. Thanks!